src/test/java/uk/gov/hmcts/reform/wataskmonitor/services/jobs/termination/TerminationJobServiceTest.java

```diff
@@ -107,8 +126,13 @@ void should_succeed_when_no_tasks_returned() throws JSONException {
     }
 
```

Comment:
do we need a test to show that active tasks are selected now fall within the last hour or 2? i.e. the new configuration is working with finishedAfter filter.

Reply:
Sadly, we can't test this through functional tests because we can't modify the details of the jobs created through testing and also there are competing jobs running in different env that can interfere with the tests created in your tests. I have discussed with Ravi and Alex and we agreed to validate the query.